server/src/test/java/org/elasticsearch/index/shard/PrimaryReplicaSyncerTests.java

```diff
@@ -76,7 +76,7 @@ public void testSyncerSendsOffCorrectDocuments() throws Exception {
             // Index doc but not advance local checkpoint.
             shard.applyIndexOperationOnPrimary(Versions.MATCH_ANY, VersionType.INTERNAL,
                 SourceToParse.source(shard.shardId().getIndexName(), "_doc", Integer.toString(i), new BytesArray("{}"), XContentType.JSON),
-                IndexRequest.UNSET_AUTO_GENERATED_TIMESTAMP, false);
+                randomNonNegativeLong(), randomBoolean());
```

Comment:
I think we randomly want to have normal ops here (i.e. UNSET_AUTO_GENERATED_TIMESTAMP)